tokens/0x2a1dbabe65c595b0022e75208c34014139d5d357.yaml

```diff
@@ -0,0 +1,15 @@
+---
+addr: '0x2a1dbabe65c595b0022e75208c34014139d5d357'
+decimals: 18
+description: >-
+  A patient and doctor-centric health ecosystem in specialised medicine engineered on blockchain.
+links:
+- Bitcointalk: https://bitcointalk.org/index.php?topic=2822113.0
+- Blog: https://medium.com/trustedhealth-io
+- Linkedin: https://www.linkedin.com/company/trustedhealth-io/
+- Reddit: 'https://www.reddit.com/r/TrustedHealth/ '
```

Comment:
This line is causing a check error: 
```
  2) Failure/Error: AssertionError: expected Reddit link https://www.reddit.com/r/TrustedHealth/  to be active, but got 501
        in test_http_link_active: link URL must be active
```

This URL is invalid because it has an extra space at the end. The space (and the `'` around the URL) should be removed.